src/components/victory-pie.js

```diff
@@ -267,7 +268,22 @@ export default class VictoryPie extends React.Component {
       CustomPropTypes.allOfType([CustomPropTypes.integer, CustomPropTypes.nonNegative]),
       PropTypes.string,
       PropTypes.arrayOf(PropTypes.string)
-    ])
+    ]),
+    /**
+     * The containerComponent prop takes an entire component which will be used to surround
```

Comment:
this wording is a little clunky... maybe "The containerComponent prop takes an entire component which will be used to create a container element for standalone charts. The new element created from the passed containerComponent will be provided with ... props... provided by VictoryPie. Something about title and desc... caveats about overridden props... "